### DISCUSSION TITLE
WarmUp needed for console commands.

### DISCUSSION THREAD

**Discussion by Nesseley** [Zrips/CMI#4230] [Feature Requests]
**Description of feature/improvement request:**
I'm making a teleport-scroll-item.
This is what the item has bound to it: 
`/cmi attachcommand !limiteduse:1!;;!right!!cc!cmi spawn [playerName];;!left!!cc!cmi spawn [playerName]`

Problem is, I want a WarmUp before the player get teleported, (so that they cannot cheat death by teleporting to spawn) and I don't seem to be able to use the config to add a warmup that is run from the console only when the player itself runs the /cmi spawn command a warmup is added.

I tried adding these and none worked. I'm a bit lost.
```
  List:
  - cc! cmi spawn:10:false:GlyphHead
  - cmi spawn [playerName]:5:false:GlyphHead
  - cmi spawn :5:false:GlyphHead
```

Edit to add: I cannot run a sudo from console to make the player run the command himself. As he then NEEDS the perm to perform the command. And I do not want my players to be able to use /spawn without buying and using the item itself.

---
### Detailed explanation of what it should do, when and in what situation: ###
Add ability to define warmup from console in config.yml

### Basic explanation as to why this feature is needed and/or useful to have: ###
Block players from using teleport-items to cheat death and teleport to spawn in an instant.

**Comment by broken1arrow** [comment]
for spawn command, you should set a delay (but seams not work).

But you can set custom alias like this. player use /spawn1 or set up a item with /spawn1 command. 
 spawn1:
    Cmds:
    - delay! 3
    - asConsole! cmi spawn $1

Edit delay! 3 give 3 seconds.
